src/AppBundle/Controller/UserController.php

```diff
@@ -161,7 +161,8 @@ public function AdminEditAction (User $user, Request $request)
      */
     public function DeleteUserAction(User $user)
     {
-        if ($this->getUser()->getRoles() == ["ROLE_ADMIN"] && $user->getRoles() == ["ROLE_SUPER_ADMIN"])
+        if (in_array('ROLE_ADMIN', $this->getUser()->getRoles()) &&
+            in_array('ROLE_SUPER_ADMIN', $user->getRoles()))
```

Comment:
Are `$this->getUser()->getRoles()` and `$user->getRoles()` the same ? If so, stick to one.

Reply:
je compare entre le rôle du user connecté et le rôle du user à delete